client/src/components/Login.js

```diff
@@ -55,113 +56,149 @@ const useStyles = makeStyles(theme => ({
 function Login() {
   const classes = useStyles();
 
-  return (
-    <Grid container component="main" xs={12} sm={8} className={classes.root}>
-      <CssBaseline />
-      <Grid
-        item
-        xs={false}
-        sm={6}
-        md={5}
-        className={classes.image}
-        style={{ display: "flex", justifyContent: "flex-start" }}
-      >
-        <div
-          style={{
-            display: "flex",
-            alignItems: "center",
-            justifyContent: "center",
-            flexDirection: "column",
-            height: "150px"
-          }}
+  const [email, setEmail] = useState("");
+  const [password, setPassword] = useState("");
+  const [error, setError] = useState(null);
+  const handleChange = event => {
+    if (event.target.name === "email") {
+      setEmail(event.target.value);
+    }
+    if (event.target.name === "password") {
+      setPassword(event.target.value);
+    }
+  };
+  const signIn = () => {
+    signApi(email, password)
+      .then(data => {
+        const token = data.token;
+        // console.log(token);
```

Comment:
Can you remove the `console.log`, it's not a good practice to leave it on the code ;)